docs/administrator.md

```diff
@@ -617,6 +617,8 @@ of the backup cron job.
 `cronjobs` resource from the `batch` API group for the operator service account.
 See [example RBAC](../manifests/operator-service-account-rbac.yaml)
 
+You cannot use logical backups when you are cloning cluster. For that you need to setup WAL writing into S3 or GCS.
```

Comment:
Can you remove this sentence? I understand the intention behind it, but I find it rather confusing then helping. We should add a paragraph here, that we advice to use WAL archiving as backup strategy as we do not support restoring from logical backups as of now. But then we should also point to a corresponding chapter which is still lacking, unfortunately. We should change that soon.

Reply:
@FxKu done!